inform7/Internal/Inter/BasicInformExtrasKit/Sections/Miscellany.i6t

```diff
@@ -46,17 +46,6 @@ Constant ACTION_PROCESSING_RB = -1; ! Identifier needs to exist even when the ru
 	Submain();
```

Comment:
I didn't really look at how things work in Basic Inform, but I think `Main` needs to be changed to run the starting the virtual machine activity.